### PR TITLE
Scenario 4

### LINKED ISSUE #92: Scenario 4 alert
Check if request on server properly rebalanced after autoscaling by checking response time.
Response time threshold 60ms
```
- AUTOSCALING
- Status: Alerting
- Alert message: Response time are above "threshold"
- CPU utilization: current total cpu utilization
```

### PR DESCRIPTION
**Pull request by Polina-Gubina**
Close #92

### CODE REVIEW

File: delatore/sources/influx.py [hunk 1]
```diff
@@ -556,6 +556,61 @@ class InfluxSourceSFSStatus(InfluxSourceLBDOWN):
...
+                results.append(generate_error(main_metric.name, self._error_message_template.format(threshold=self._threshold, cpu_utilization=round(cpu_utilization, 2))))
+        return generate_message(self.CONFIG_ID, results)
+
+    async def _get_auxilary_metrics(self, metric):
+        query = metric.query.format(entity=metric.metric_id)
+        result = await self.influx_client.query(query)
+        try:
+            cpu_utilization = result.raw['series'][0]['values'][0][1]
+        except(IndexError, KeyError):
+            cpu_utilization = 'No data'
```
Comment (lego963) [inline delatore/sources/influx.py:590]: if 'No data' returns how will `round()`?

File: delatore/sources/influx.py [hunk 2]
```diff
@@ -556,6 +556,61 @@ class InfluxSourceSFSStatus(InfluxSourceLBDOWN):
...
+        last_time_ms = None
+        error_status = False
+        status = Status.OK
+        try:
+            response_line = last_record.raw['series'][0]['values'][0]
+        except(IndexError, KeyError):
+            status = Status.NO_DATA
+            return error_status, last_time_ms, status
+        response_time = response_line[1]
+        if response_time > self._threshold:
```
Comment (lego963) [inline delatore/sources/influx.py:605]: in _get_status we get info
in _get_update we compare info from _get_status with thresholds or other params

Comment (Polina-Gubina) [reply delatore/sources/influx.py:605]: My approach seems more convenient for me

File: delatore/sources/influx.py [hunk 3]
```diff
@@ -556,6 +556,61 @@ class InfluxSourceSFSStatus(InfluxSourceLBDOWN):
...
+        main_metric = self._metrics[0]
+        error_status, last_time_ms, status = await self._get_status(main_metric)
+        results = []
+        if status == status.FAIL:
+            results.append(generate_status(main_metric.name, status,
+                                           last_time_ms.strftime(UNIFIED_TIME_PATTERN)))
+        if status == status.NO_DATA:
+            results.append(generate_status(main_metric.name, status, None))
+        else:
+            if error_status == True:
```
Comment (lego963) [inline delatore/sources/influx.py:578]: `if error_status == True` could be simplified `if error_status:`

**Pull request merged**